a-tribute-page/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>A Tribute Page</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+    <body>
+        <div>
+            <header>
+                <h1>Mans Sadykov</h1>
+                <p>1997-20..</p>
+            </header>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat quasi voluptas asperiores! Porro possimus pariatur harum deleniti facilis? Esse quos accusamus maiores delectus tenetur perferendis labore recusandae doloribus eaque voluptas?</p>
```

Comment:
There must be a div as a parent. Fix that

Reply:
Fixed , check